news/2019-04-22-project-loved-week-of-april-21st.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+title: "Project Loved: Week of April 21st"
+date: 2019-04-22 18:00:00 +0000
+---
+
+[Project Loved](https://osu.ppy.sh/community/forums/120)'s cycle continues! The captains have nominated a new batch of maps for the Loved section, but it's only up to you whether or not they'll make it! Come check out the selection and vote for the maps that you'd like to see Loved.
+
+![](/wiki/shared/news/banners/project-loved.jpg)
+
+Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! Every two weeks, we pick out a few beatmaps from each game mode that will get their shot at making it into the [Loved category](https://osu.ppy.sh/beatmapsets?s=8). From the maps we selected, you can now vote whether or not you want them to be Loved! Click on each map's header image in this post to be taken to its voting thread.
+
+Our lineup of captains has changed for osu!catch again, this time with [Ascendance](https://osu.ppy.sh/users/2931883) stepping down to make room for his new responsibilities as an NAT member, and [Yumeno Himiko](https://osu.ppy.sh/users/1806962) filling their shoes. Say your thanks and welcomes, and wish both of them the best of luck!
```

Comment:
as a NAT member

Reply:
had to look this one up while I was writing it https://www.google.com/search?&q=a+or+an+before+acronym

I think NAT is read letter-by-letter so "an" is better (since N is like "enn")?